lib/prefix_api_key/generate.rb

```diff
@@ -39,5 +34,9 @@ def password_short
     def password_long
       PrefixApiKey.configuration.password_short
     end
+
+    def generate_token(prefix)
+      "#{prefix}_#{generate_short_token(prefix)}_#{generate_long_token(prefix)}"
```

Comment:
Hello @italomatos, thanks for the contribution.

I think the generate_short_token and generate_long_token methods were missing in the PR